build.gradle

```diff
@@ -9,8 +9,7 @@ plugins {
   id "org.sonarqube" version "3.2.0"
 }
 
-// TODO: Update group to end with "admin" or "trainee".
-group = "uk.nhs.hee.tis"
+group = "uk.nhs.hee.tis.revalidation.migration"
```

Comment:
Should the group = "uk.nhs.hee.tis.revalidation" ?

Reply:
Done